### ISSUE TITLE
All pages missing a skip-to-content link

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#825]
## What
[WCAG's Bypass Blocks criterion](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) requires mechanisms to "bypass blocks of content that repeated on multiple pages". Examples of this are site headers and "global" navigation bars.

A common way to achieve this is to provide a "skip" link, which is the first link in a page's DOM and is hidden from view but accessible to keyboard and assistive technology users. Examples of sites with skip links are [Microsoft](https://www.microsoft.com/en-us/), [Bootstrap](https://getbootstrap.com), [The Guardian](https://www.theguardian.com/), and [24 Ways](https://24ways.org).

## How
1. Create a link directly after the `<body>` element.
2. Hide it visually using whatever equivalent we have to Bootstrap's `sr-only` class.
3. Make sure that the link points to the `<main>` element.

This is currently blocked by [issue 779](https://github.com/hackoregon/civic/issues/779)
